package/yast2-security.changes

```diff
@@ -1,3 +1,11 @@
+-------------------------------------------------------------------
+Fri Jan  7 12:37:08 UTC 2022 - Knut Anderssen <kanderssen@suse.com>
+
+- Do not use the 'lsm' kernel boot parameter by now as it could
+  need other modules to be pecified like the integrity one
```

Comment:
typo: s/pecified/specified/

Reply:
Thanks @pevik 

I caught the typo this morning too, but the change was sent a month ago or so.